CONTRIBUTORS.md

```diff
@@ -154,6 +154,9 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for details on how to contribute to Cesiu
   - [Jonathan Nogueira](https://github.com/LuminousPath)
 - [Palantir Technologies, Inc.](https://palantir.com)
   - [Joey Rafidi](https://github.com/jrafidi)
+- [Geoplex GIS GmbH](https://www.geoplex.de)
+  - [Hannes Gräuler](https://github.com/lordi)
```

Comment:
It looks like this PR only has commits from `LHolst`.  Should this line be removed?